concourse/pipelines/destroy.yml

```diff
@@ -81,7 +80,12 @@ jobs:
 
       - task: vpc-terraform-outputs-to-sh
         config:
-          image: docker:///ruby#2.2-slim
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: governmentpaas/git-ssh
+              tag: "2.2-slim"
```

Comment:
I don't think this was meant to be updated to use the `git-ssh` container. There's no `2.2-slim` tag for that container. (Fix in #51)